packages/avatars/demo/avatars.stories.js

```diff
@@ -46,72 +58,100 @@ export const Types = {
 
       options: [undefined, 'c-avatar--xs', 'c-avatar--sm', 'c-avatar--lg']
     }
-  }
+  },
+  decorators: [Story => <Story />]
```

Comment:
Unable to figure out why, but this is needed to bring in the global `ThemeProvider` decorator 😞